websites/B/BuildTheEarth/dist/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+      "name": "fantomitechno 🦊",
+      "id": "563749920683720709"
+    },
+    "url": "buildtheearth.net",
+    "description": {
+      "en": "We are recreating the entire planet in Minecraft."
+    },
+    "service": "BuildTheEarth",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/XvapvDX.png",
+    "thumbnail": "https://i.imgur.com/YzqC7ju.png",
+    "color": "#104D92",
+    "tags": [
+      "minecraft",
+      "buildtheearth"
+    ],
+    "category": "games",
+    "settings": [
+      {
+        "id": "privacy",
+        "title": "Privacy mode",
+        "icon": "fad fa-user-secret",
+        "value": false
+      },
+      {
+        "id": "sprivacy",
```

Comment:
please give the ID a better name
something like mappos or something along those lines